jena-fuseki2/jena-fuseki-ui/src/views/dataset/Upload.vue

```diff
@@ -186,9 +186,9 @@
 </template>
 
 <script>
-import Menu from '@/components/dataset/Menu'
+import Menu from '@/components/dataset/Menu.vue'
 import FileUpload from 'vue-upload-component'
-import JenaTable from '@/components/dataset/JenaTable'
+import JenaTable from '@/components/dataset/JenaTable.vue'
```

Comment:
Vite does not try to automatic guess what's the extension of imports. So we need to specify it.